pkg/kube/controllers/quarksjob/errand_reconciler_test.go

```diff
@@ -160,8 +160,7 @@ var _ = Describe("ErrandReconciler", func() {
 
 				It("should log create error and requeue", func() {
 					_, err := act()
-					Expect(logs.FilterMessageSnippet(fmt.Sprintf("Failed to create job '%s': could not create service account for "+
-						"pod in qJob '%s': fake-error", qJobName, qJobName)).Len()).To(Equal(1))
+					Expect(logs.FilterMessageSnippet(fmt.Sprintf("Failed to create job '%s': could not create service account: fake-error", qJobName)).Len()).To(Equal(1))
```

Comment:
😂may it's a line-wrapping.

Reply:
The error message also changed so slightly. My editor doesn't break strings with `+` automatically and it also looks strange.. :) :)

![wrapping](https://user-images.githubusercontent.com/393653/69327833-764d9600-0c4e-11ea-91d0-c241591f74f0.png)